cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -261,6 +519,8 @@ public String getFormat() {
     public static final String EXPORT_POWER_FLOWS_FOR_SWITCHES = "iidm.export.cgmes.export-power-flows-for-switches";
     public static final String NAMING_STRATEGY = "iidm.export.cgmes.naming-strategy";
     public static final String PROFILES = "iidm.export.cgmes.profiles";
+    public static final String CGM_EXPORT = "iidm.export.cgmes.cgm_export";
+    public static final String MODEL_UPDATE = "iidm.export.cgmes.model_update";
```

Comment:
This parameter key is not used anymore (unless in some unit tests).

Reply:
removed the obsolete parameter name and its uses in unit tests